tests/fixtures/long.md

```diff
@@ -17,16 +17,33 @@ function foo() {
 }
 ```
 
+<!-- eslint-env node -->
+<!-- eslint-disable eol-last, quotes -->
+
 ```node
 console.log(process.version);
 ```
 
 How about some JSX?
 
+<!--
+    eslint quotes: [
+        "error",
+        "single"
+    ]
+-->
+<!--eslint-disable no-console-->
+
 ```jsx
 console.log("Error!");
 ```
 
     I may be a code block, but don't lint me!
 
+<!-- eslint-disable -->
```

Comment:
Correct. I should document that.

    $ cat pull55.md
    <!-- eslint-disable -->
    
    ```js
    alert('Hello, world!')
    ```
    
    ```js
    alert('Hello, world!')
    ```
    $ node_modules/.bin/eslint pull55.md
    
    /Users/brandon/code/eslint/eslint-plugin-markdown/pull55.md
      8:1   error  Unexpected alert              no-alert
      8:1   error  'alert' is not defined        no-undef
      8:7   error  Strings must use doublequote  quotes
      8:23  error  Missing semicolon             semi
    
    ✖ 4 problems (4 errors, 0 warnings)